More/EAV/Attribute.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace DesignPatterns\More\EAV;
+
+/**
+ * Class Attribute
+ */
+class Attribute implements ValueAccessInterface
+{
+    /**
+     * @var array|Value[]|ValueInterface[]
```

Comment:
Would it not be clearer here to have the interface type `ValueInterface[]` rather than 3? As this could grow as more **concrete** classes are added